src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -36,10 +36,15 @@
  */
 final class ClassAttributesSeparationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
 {
+    const SPACING_ONE = 'one';
+    const SPACING_NONE = 'none';
+
     /**
      * @var array<string, true>
      */
     private $classElementTypes = [];
+    private static $supportedSpacings = [self::SPACING_NONE, self::SPACING_ONE];
+    private static $supportedTypes = ['const', 'property', 'method'];
```

Comment:
Let's have alphabetical order of the elements (especially as they are used in alphabetical order in the description).